scripts/openstack_compute_install.sh

```diff
@@ -11,4 +11,12 @@ salt -C 'I@opencontrail:control:id:1' cmd.run "/usr/share/contrail-utils/provisi
 # Reboot compute nodes
 salt "cmp*" system.reboot
 
-sleep 30
+while true; do
+	salt '*' test.ping | grep -q Minion
+	if [ $? -ne 0 ]; then
+		break
+	fi
```

Comment:
Just tried it but it fails:

root@cfg01:~# salt 'cmp02*' system.reboot
root@cfg01:~# if salt 'cmp*' test.ping ; then echo ok; else echo ko; fi
cmp01.mk22-lab-advanced.local:
    True
cmp02.mk22-lab-advanced.local:
    Minion did not return. [Not connected]
ok
root@cfg01:~# sleep 180
root@cfg01:~# if salt 'cmp*' test.ping ; then echo ok; else echo ko; fi
cmp01.mk22-lab-advanced.local:
    True
cmp02.mk22-lab-advanced.local:
    True
ok

I can however do something like

if ! salt 'cmp*' test.ping --out txt | grep -q Minion; then break; fi